src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,45 @@
+package mate.controller;
```

Comment:
Let's create a separate packages `driver`, `car`, `manufacturer` inside `controller` package and divide controllers between these packages